.github/workflows/ci_rust.yml

```diff
@@ -45,7 +45,8 @@ jobs:
 
       - name: Tests
         working-directory: ${{env.ROOT_PATH}}
-        run: cargo test --all-features
+        # Test all features except for FIPS, which is tested separately.
+        run: cargo test --features unstable-fingerprint,unstable-ktls,quic,pq,testing
```

Comment:
Yeah I agree that it's unfortunate to not have this update automatically. I looked for a way to do this but couldn't find anything. I did find a discussion about it though: https://internals.rust-lang.org/t/pre-rfc-cargo-test-default-features-except-and-co/20850

I did add a new test that runs all features without doc comments, though. So worst case if someone forgets to add their feature to this list, it will still be tested, it just won't test doc comments that only exist when the feature is enabled I think.

Reply:
Alternatively I could just disable my doc comments with an AWS-LC dependency, if that seems better.